README-unspun.md

```diff
@@ -0,0 +1,36 @@
+# Changes from mainline
+1. Add CMake build support
+
+# CMake build support
+## How to use lib
+1. Set the config folder using the FREERTOS_CONFIG_FOLDER variable.
+Example:
+```
+set(FREERTOS_CONFIG_FOLDER ${CMAKE_CURRENT_SOURCE_DIR})
+```
+2. Add the subdirectory.
+Example:
+```
+add_subdirectory(../ freertos)
+```
+
+Also see the test_cmake folder for working example. 
+
+## Supported compiler/arch configs
+- GCC/ARM_CM4F
+
+## Run the CMake build tests
+### Pre-Reqs
+1. ARM GCC tool chain is installed and has been added to the PATH
+2. Make is installed and has been added to the PATH
```

Comment:
When I run this from a stock ubuntu container, I have to explicitly install cmake. Should that be called out here or am I doing something wrong?

Reply:
Yes, I had to do that as well, I'll add that here!